test/zod/test_zod.ts

```
@@ -38,3 +38,21 @@
   };
   assert.deepStrictEqual(expected, data);
 });
+
+test("test zod image path", async () => {
+  const data = mediaSourceSchema.safeParse({kind: "path", path: ""});
```

ESLint

[error] Replace `kind:·"path",·path:·""` with `·kind:·"path",·path:·""·`

(prettier/prettier)